web/src/views/DeviceRegistration/RegisterOneTimePassword.tsx

```diff
@@ -49,16 +51,19 @@ const RegisterOneTimePassword = function () {
             console.error(err);
             if ((err as Error).message.includes("Request failed with status code 403")) {
                 createErrorNotification(
-                    "You must open the link from the same device and browser that initiated the registration process",
+                    t(
+                        "You must open the link from the same device and browser that initiated the registration process",
+                    ),
                 );
             } else {
                 createErrorNotification(
-                    "Failed to register device, the provided link is expired or has already been used",
+                    t("Failed to register device, the provided link is expired or has already been used"),
                 );
             }
             setHasErrored(true);
         }
         setIsLoading(false);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [processToken, createErrorNotification]);
```

Comment:
```suggestion
    }, [processToken, createErrorNotification, t]);
```

Reply:
should i remove this line (`// eslint-disable-next-line react-hooks/exhaustive-deps`) 
that line was added to prevent lint warnings

Reply:
Check out the addition in the second line, that line are the exhaustive deps for the react hook.

Reply:
done